--- a/src/Audience.vala
+++ b/src/Audience.vala
@@ -37,7 +37,7 @@
          */
         /// TRANSLATORS: This is the name of the application shown in the application launcher. Some distributors (e.g. elementary OS) choose to display it instead of the brand name "Audience".
         public const string VIDEOS = N_("Videos");
-        public const string PROGRAM_NAME = "Audience";
+        public const string PROGRAM_NAME = VIDEOS;
         /// TRANSLATORS: These are the keywords used when searching for this application in an application store or launcher.
         public const string KEYWORDS = N_("Audience;Video;Player;Movies;");
         public const string COMMENT = N_("Watch videos and movies");
--- a/data/audience.desktop
+++ b/data/audience.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Version=1.0
-Name=Audience
+Name=Videos
 GenericName=Video Player
 Comment=Watch videos and movies
 Exec=audience %U
@@ -16,4 +16,4 @@
 
 [Desktop Action AboutDialog]
 Exec=audience --about
-Name=About Audience
+Name=About Videos
