--- a/src/Audience.vala
+++ b/src/Audience.vala
@@ -177,7 +177,7 @@
             mainwindow.events |= Gdk.EventMask.POINTER_MOTION_MASK;
             mainwindow.events |= Gdk.EventMask.LEAVE_NOTIFY_MASK;
             mainwindow.events |= Gdk.EventMask.BUTTON_PRESS_MASK;
-            mainwindow.title = program_name;
+            mainwindow.title = _("Videos");
             mainwindow.window_position = Gtk.WindowPosition.CENTER;
             mainwindow.set_application (this);
             mainwindow.add (mainbox);
--- a/data/audience.desktop
+++ b/data/audience.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Version=1.0
-Name=Audience
+Name=Videos
 GenericName=Video Player
 Comment=Watch videos and movies
 Exec=audience %U
@@ -16,4 +16,4 @@
 
 [Desktop Action AboutDialog]
 Exec=audience --about
-Name=About Audience
+Name=About Videos
