--- a/data/audience.desktop
+++ b/data/audience.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Version=1.0
-Name=Audience
+Name=Videos
 GenericName=Video Player
 Comment=Watch videos and movies
 Exec=audience %U
@@ -16,4 +16,4 @@
 
 [Desktop Action AboutDialog]
 Exec=audience --about
-Name=About Audience
+Name=About Videos
--- a/src/Audience.vala
+++ b/src/Audience.vala
@@ -52,7 +52,7 @@
         public const string ABOUT_GENERIC = N_("About Videos");
 
         construct {
-            program_name = "Audience";
+            program_name = _("Videos");
             exec_name = "audience";
 
             build_data_dir = Constants.DATADIR;
